ui/js/auth/LoginCallback.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import Icon from 'react-fontawesome';
+
 import AuthService from './AuthService';
 import { webAuth, parseHash } from './auth-utils';
 
-export default class LoginCallback extends React.PureComponent {
```

Comment:
The changes in this file technically aren't required, but make the file more consistent with the others, and reduce the size of the Neutrino 9 PR.

Reply:
Ahh, yeah.  This was just cruft I missed.  Thanks!  :)

Reply:
(For Neutrino 9 we also need to switch to react-hot-loader v4, which has a different way of setting it up, so I need to add the `hot(module)...` bit here, which is easier if the export is separate)